packages/react-native-codegen/src/parsers/typescript/components/events.js

```diff
@@ -236,7 +237,7 @@ function buildEventSchema(
     findEventArgumentsAndType(parser, typeAnnotation, types);
 
   if (!argumentProps) {
-    throw new Error(`Unable to determine event arguments for "${name}"`);
+    throwIfArgumentPropsAreNull(argumentProps, name);
   } else if (!bubblingType) {
     throwIfBubblingTypeIsNull(bubblingType, name);
   } else {
```

Comment:
So, the goal is to remove the `if-else` as Dmitry suggested. 🤔 

one way could be to:
1. Define a type alias like `type ArgumentProps = $FlowFixMe`
2. Have `throwIfArgumentPropsAreNull` return `ArgumentProps` (not nullable)
3. invoke it like `argumentProps = throwIfArgumentPropsAreNull(argumentProps, name);`

A bit ugly, but it should do the trick.

Otherwise, we can actually suppress the warning with a `// eslint-disable-next-line <name of the broken rule>` as we are actually sure that the props can't be null now.

I'm okay with both approaches!

Reply:
I chose the first option. Even though I don't like having to make the code more complex for typing reasons, I think it's safer.